### PR TITLE
added strings for the health certificate details

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
added strings for the health certificate

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7211

### CODE REVIEW

**Review by ArturFriesen — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -2467,3 +2467,25 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
 
 "HealthCertificate_Info_disclaimer" = "Ausführliche Informationen zur Datenverarbeitung und Ihrem Einverständnis.";
+
+/* Health Certificate Details */
+
+"HealthCertificate_Details_vaccinationCount" = "Impfung %i von %i";
+
+"HealthCertificate_Details_certificate" = "Impfzertifikat";
+
+"HealthCertificate_Details_dateOfBirth" = "geboren %@";
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2477]: is there missing "am"? 
So that the value would be "geboren am %@"?

**Review by pascal-brause — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:2477]: No - this is what was given. I guess it will change later

**Review by naveeddotio — APPROVED** [review]

**Comment by naveeddotio** [comment]
@NataliaLemmerth

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

**Pull request merged**